packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -93,59 +102,65 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
         },
         carousel__themeDark: {
             "& $carousel_flipperPrevious, & $carousel_flipperNext": {
-                color: darkGray,
-                fill: darkGray,
-                background: "rgba(255, 255, 255, 0.6)",
-                border: `1px solid ${lightGrayRest}`,
+                "&::before": {
+                    color: neutralForegroundRest((): string => black),
+                    fill: neutralForegroundRest((): string => black),
+                    background: neutralFillStealthRest((): string => black),
+                    border: `1px solid ${neutralOutlineRest((): string => black)}`,
```

Comment:
I'm wondering about these default values. Is there a case where the design system may not provide values? If so, seems like we would want the fallback universally, not in each case. Also, everything can't be the same color or it would just be a black circle.

Reply:
I'm not following. Black is passed a reference color to keep the Dark Themed slide flippers always dark. We are three different recipes here `neutralForegroundRest` `neutralFillStealthRest` and `neutralOutlineRest`

Reply:
This code merges the provided `DesignSystem` with a default `DesignSystem`, so these values will always exist.